sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/cosmos/CosmosHealthConfiguration.java

```diff
@@ -16,15 +16,15 @@
  * CosmosHealthConfiguration
  */
 @Configuration
-@ConditionalOnClass({ CosmosClient.class, HealthIndicator.class})
+@ConditionalOnClass({ CosmosAsyncClient.class, HealthIndicator.class})
```

Comment:
Why do we change to use async client instead?

Reply:
there's a new bean cosmosAsyncClient instead of cosmosClient in `AbstractCosmosConfiguration` class, just follow the spring-cosmos.
```java`
    /**
     * Declare CosmosAsyncClient bean.
     *
     * @param cosmosClientBuilder cosmosClientBuilder
     * @return CosmosAsyncClient bean
     */
    @Bean
    public CosmosAsyncClient cosmosAsyncClient(CosmosClientBuilder cosmosClientBuilder) {
        return CosmosFactory.createCosmosAsyncClient(cosmosClientBuilder);
    }
```